BB-CEST/BB_RemotiveCloud

```diff
@@ -0,0 +1,57 @@
+Name 
+RemotiveCloud
+
+BB Tags(s) 
+BB-CEST; BB-CSC-TC
+
+Functional Cluster(s) (optional)	
+Diagnostics (could also be used for feeding vehicle data to emulators and other prototyping tools)
+
+Layer	
+Cloud
```

Comment:
Is it a plain Cloud Application or there are also Cloud Middleware Services included which can by for example re-used by other Cloud App Developers?

Reply:
I think we can resolve this discussion.